src/lua/httpc.lua

```diff
@@ -228,6 +241,56 @@ local function process_headers(headers)
     return headers
 end
 
+-- In the Extended BNF notation of RFC 1341, a Content-Type header field value
+-- is defined as follows: Content-Type := type "/" subtype *[";" parameter].
+-- See RFC 1341, https://www.w3.org/Protocols/rfc1341/4_Content-Type.html
+-- Function splits content-type header for part with type and subtype and part
+-- parameter and returns the first one.
+local function header_value_wo_parameters(header_value)
```

Comment:
I don't like several things in this function name:

* It is specific for content-type header, but named with the 'header' word like it is common for all headers. IOW, not 'content-type value without parameters', but 'header value without parameters'.
* It contains abbreviation 'wo'.
* It contains the common word 'value', which don't add much information.
* It says about parameters but not what we get without them.

Variants:

* `extract_mime_type()` (if you'll choose `mime_type` term for `type/subtype` part of the content-type).
* Just `mime_type()` if it'll not clash with a variable name (called inline).
* Or, if you'll choose another term for `type/subtype`, `extract_<your_term>()`.